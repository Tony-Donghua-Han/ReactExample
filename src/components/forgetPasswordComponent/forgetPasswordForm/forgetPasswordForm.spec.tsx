import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import ForgetPasswordForm from "./forgetPasswordForm";

describe("ForgetPasswordForm", (): void => {
    let wrapper: ShallowWrapper;
    beforeEach((): void => {
        wrapper = shallow(<ForgetPasswordForm onSubmit={(): void => {}} />).dive();
    });

    test("should render without crushing", (): void => {
        expect(wrapper.debug()).toMatchSnapshot();
    });
});
