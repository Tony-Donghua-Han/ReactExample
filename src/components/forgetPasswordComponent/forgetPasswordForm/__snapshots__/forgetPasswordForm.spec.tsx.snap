// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgetPasswordForm should render without crushing 1`] = `
"<ContextProvider value={{...}}>
  <Form className=\\"forgetPasswordForm\\" id=\\"forgetPasswordForm\\">
    <FormikConnect(FieldInner) type=\\"input\\" name=\\"email\\" className=\\"inputBox\\" placeholder=\\"Email Address\\" />
    <FormikConnect(ErrorMessageImpl) name=\\"email\\" className=\\"alert\\" component=\\"div\\" />
    <button type=\\"submit\\" disabled={false} className=\\"submitButton\\">
      Reset My Password
    </button>
  </Form>
</ContextProvider>"
`;
